src/middleware/userExists.js

```diff
@@ -0,0 +1,23 @@
+import models from '../sequelize/models';
+
+const checkEmail = {
+  async userExist(req, res, next) {
+    const { email } = req.body;
```

Comment:
You can create several functions that validate user existence and other validations like body and export them instead of creating several files.